docs/content/cli-arguments.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Ingress Controller Command-line Arguments
+
+description: 
+weight: 100
+doctypes: ["concept"]
+toc: true
+---
+
+
```

Comment:
We can remove this doc. We'll add its path as an alias in the new doc, so users will be redirected automatically.

Reply:
Hugo aliases: https://gohugo.io/content-management/urls/#aliases

If that doesn't work, we can always define redirects in the netlify.toml file.

Reply:
That sounds good.  Where should I add the alias?

Reply:
Okay.  This one has been removed.  Should I create the new file or will that be done later?

Reply:
I got the impression from the comment that had already been done. I think we need the team to weigh in on that. 

@pleshakov ??